redhat-developer/developers.redhat.com
# [RHDENG-2486] Install "Read only mode" module for Drupal

This PR installs the [Read only mode](https://www.drupal.org/project/readonlymode) module for Drupal.

As part of deployments onto Managed Platform, we will be putting Drupal into read-only mode so that content cannot be created whilst the new version of Drupal is deployed. This module prevents new content from being created in Drupal, but still allows the site to function as normal for anonymous visitors so that the site can continue to function whilst a deployment is on-going.

### JIRA Issue Link
* [RHDENG-2673](https://issues.jboss.org/browse/RHDENG-2673)

### Verification Process

* The entire build should go green
* Visit `admin/config/development/maintenance` in the Drupal admin interface
* Ensure that there is now an option titled `Read Only Mode`
* Ensure that the Read Only Mode is disabled by default
* Visit `admin/config/development/configuration` in the Drupal admin interface and verify that there are no config changes left to import.
